android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt

```diff
@@ -51,18 +79,22 @@ class CommandsOnLaunch {
         assertThat(addresses[0].receivers.sapling).isEqualTo("zs16uhd4mux24se6wkm74vld0ec63d4dxt3d7m80l5xytreplkkllrrf9c7fj859mhp8tkcq9hxfvj")
         assertThat(addresses[0].receivers.orchard_exists).isEqualTo(true)
     }
+}
 
+@Category(OfflineTest::class)
+class ExecuteAddressesFromUfvk {
     @Test
     fun executeAddressFromUfvk() {
         val mapper = jacksonObjectMapper()
 
         val server = "https://mainnet.lightwalletd.com:9067"
```

Comment:
Here the server could be `http://10.0.2.2:20000` ????

Reply:
That's stupid of me... sorry.

Reply:
this is exactly what i have now implemented so not stupid! this makes the `offline` tests actually offline as in these tests cargo does not setup regtest and its pointed at main net for entropy bug tests etc.